buildconfig/Jenkins/Conda/conda-buildscript-linux

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash -ex
```

Comment:
I like all of the docstrings in these scripts. They are very useful as a giude to how they are intended be used. Nice. 👍🏻

Reply:
Thanks! without them I think it would be horribly unclear what you need to do.